packages/next-codemod/bin/upgrade.ts

```diff
@@ -125,6 +125,10 @@ export async function runUpgrade(
     await runTransform(codemod, process.cwd(), { force: true })
   }
 
+  if (targetReactVersion.startsWith('19')) {
```

Comment:
This also covers React 19 -> React 19.

We should just reuse the logic from choosing codemods. React 19 was in canary.45 or something (please double check).